rpc/src/main/java/io/teris/rpc/ServiceDispatcherImpl.java

```diff
@@ -46,6 +50,8 @@
 
 		private ExecutorService executors = null;
 
+		private Supplier<String> uidGenerator = () -> UUID.randomUUID().toString();;
```

Comment:
2 ;;

Reply:
uuid is an implementation detail, this can be any unique id